concepticondata/conceptlists/Bowern-2017-348.tsv

```diff
@@ -311,12 +311,12 @@ NUMBER	ID	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	COMMENTS
 310	Bowern-2012-348-310	tomorrow	1329	TOMORROW	
 311	Bowern-2012-348-311	tongue	1205	TONGUE	
 312	Bowern-2012-348-312	tooth	1380	TOOTH	
-313	Bowern-2012-348-313	top grinding stone	3027	TOP GRINDSTONE	
+313	Bowern-2012-348-313	top grinding stone	2459	GRINDSTONE	
```

Comment:
sorry for my ignorance, but if there's a bottom grindstone, why would we not keep it as a concept? Mapping to a broader concept in this case seems unnecessary, since you can also handled it via the conceptrelations.tsv

Reply:
I am really in doubt about this; what I did was look for translations in some languages and try to infer what would be better as "default" (top or bottom grindstone). It *seems* that it is more common to call the top one as "grindstone", with second something like "stone/plane/base/wheel" plus some attribute, but it is really a wild guess. To be clear, I understand it refers to this:

![grindstone](http://www.yourlifechoices.com.au/assets/images/_thumbs550/ancient_grindstone_used_to_crush_grain_website.jpg)

My suggestion would now be to add no narrower grindstone concept, map Bowern's "top grindstone" to GRINDSTONE, and leave "bottom grindstone" unmapped. What do you think?

Reply:
well, assuming that if ever we are going to make a study on grindstones in the world, we will more likely include the normal grindstone in our concept list than the top one. Yes, leave it unmapped, so we keep the order, and if people are unhappy, they can refine in twenty years, when grindstone linguistics becomes a hype...